configurations/performance/latency-decorator-error-thresholds-steps-ent-tablets.yaml

```diff
@@ -13,7 +13,7 @@ latency_decorator_error_thresholds:
       P90 read:
         fixed_limit: 1
       P99 read:
-        fixed_limit: 1
+        fixed_limit: 3
```

Comment:
see this number is greater than for 300k - due it's first run and Scylla's cache was not fully populated